netbox_netisp/views.py

```diff
@@ -110,7 +109,16 @@ class AccountListView(ObjectListView, View):
 class AccountEditView(ObjectEditView, View):
     queryset = Account.objects.all()
     model_form = forms.AccountForm
+    def get(self, request, *args, **kwargs):
+        print(kwargs)
+        if "customer_pk" in kwargs:
+            account = Account()
+            customer = Customer.objects.get(pk=kwargs["customer_pk"]) #get current customer we're talking about
+            account.primary_applicant = customer
+            account.save() #saves to db
+            return redirect(account)
 
+        return super().get(request, *args, **kwargs)
```

Comment:
The logic flow here is not very clean. You are using an implicit else and I would prefer an explicit. Please refactor the code to be 'if customer pk then redirect else return super'